tests/tck/features/insert/InsertIfNotExists.feature

```diff
@@ -168,3 +168,110 @@ Feature: Insert vertex and edge with if not exists
     Then the result should be, in any order:
       | like._src | like._dst | like._rank | like.likeness |
       | 'Tom'     | 'Peter'   | 0          | 83            |
+
+  Scenario: insert vertex and edge with default propNames
+    Given an empty graph
+    And create a space with following options:
+      | partition_num  | 9                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    And having executed:
+      """
+      CREATE TAG IF NOT EXISTS person(name string, age int);
+      CREATE EDGE IF NOT EXISTS like(likeness int);
+      CREATE TAG IF NOT EXISTS student(grade string, number int);
+      """
+    # insert vertex succeeded
+    When try to execute query:
+      """
+      INSERT VERTEX person(name, age) VALUES "Tom":("Tom", 1)
+      """
+    Then the execution should be successful
+    When try to execute query:
+      """
+      INSERT VERTEX person VALUES "Tom":("Tom", 2)
+      """
+    Then the execution should be successful
+    When try to execute query:
```

Comment:
Need not to retry when query returns errors, here should be `executing query:`

Reply:
fix